src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,10 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    public Ball getRandomBall() {
+        ColorSupplier colorSupplier = new ColorSupplier();
+        return new Ball(colorSupplier.getRandomColor(), new Random().nextInt(100));
```

Comment:
why did you create an instance of ColorSupplier, but use Random directly in the return statement? Let's keep our code consistent and use either one of those approaches. (Or create instances for both ColorSupplier and Random, or use both of them directly in return)